src/client/editor/Editor.js

```diff
@@ -1701,4 +1706,55 @@ export default class Editor {
   redo() {
     this.history.redo();
   }
+
+  async publishScene() {}
```

Comment:
one thing we probably need to figure out (maybe for next PR) is gathering attribution and potentially also origin information for all composed assets, and submitting it to the scene API. also, reminder that we need to parameterize this so it will either create or update if the scene already has a scene ID